core/src/lib/code-generation/code-builder/queued-code-builder.ts

```diff
@@ -16,15 +17,19 @@ import { DecreaseDepthStep } from './queued-code-builder/decrease-depth-step';
  */
```

Comment:
"The QueuedCodeBuilder handles the tab depth of the code."

Mittlerweile macht er noch mehr. Kannst ja noch kurz hinzufügen.
Zb:"The QueuedCodeBuilder handles the tab depth of the code, adds new lines and imports."

Und er gibt am Ende den Code von den einzelnen queue steps als String wieder oder?
Das könntest du auch noch hinzufügen dann wei? man genau wofür die Klasse da ist. Weil wenn ich nur die Doku von getResult() lese ist es mir nicht sofort klar